### PR TITLE
[EMB-239] Proxy ember apps by default, remove unused url from dicts, set server…

### PR DESCRIPTION
**Pull request by binoculars**
… urls to 192.168.168.167 default

## Purpose

* Proxy ember apps by default (vs `send_from_directory`),
* Allows Proxying to docker or non-docker ember server
* docker-sync doesn't always sync well, so restarting the container and hoping is a pain point
* After sufficient time has passed for people to switch, we can remove `send_from_directory`

## Changes

* Removes unused `url` dict key
* Adds `EMBER_DOMAIN`, defaults to localhost, and `192.168.168.167` for dockerized OSF.

## QA Notes

N/A

## Documentation

`local.py` changes are recommended, but not required.

For the ember apps running outside of docker, the port and live reload port must be set with the `yarn start` command via cli arg e.g. 

```sh
yarn start --port 4201 --live-reload-port 41954
```

or by environment variables e.g.

```sh
export PORT=4201
export EMBER_CLI_INJECT_LIVE_RELOAD_PORT=41954
yarn start --live-reload-port 41954
```

## Side Effects

N/A

## Ticket

https://openscience.atlassian.net/browse/EMB-239

### CODE REVIEW

**Comment by jamescdavis** [comment]
Proxing the Ember app is working great, but I can't get live reload to work.

**Comment by binoculars** [comment]
Apparently the environment variable isn't used consistently

https://github.com/ember-cli/ember-cli-inject-live-reload/blob/master/index.js#L24

Setting it with `--live-reload-port` works though

File: website/routes.py [hunk 1]
```diff
@@ -197,11 +202,26 @@ def ember_app(path=None):
...
 
+    if not ember_app:
+        raise HTTPError(http.NOT_FOUND)
+
+    if settings.PROXY_EMBER_APPS:
+        url = urlparse.urljoin(ember_app['server'], request.path[len(ember_app['path']):])
+        resp = requests.get(url, stream=True, timeout=EXTERNAL_EMBER_SERVER_TIMEOUT, headers={'accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8'})
+        excluded_headers = ['content-encoding', 'content-length', 'transfer-encoding', 'connection']
+        headers = [(name, value) for (name, value) in resp.raw.headers.items() if name.lower() not in excluded_headers]
+        return Response(resp.content, resp.status_code, headers)
```
Comment (jamescdavis) [inline website/routes.py]: Passing the headers through in the response was what I was missing when I tried this. I think the browser was trying to parse the .js files as HTML, which would explain why it was so slow. Nice job @binoculars and @icereval!

Comment (binoculars) [reply website/routes.py]: The express server in ember-cli tries to do a keepalive connection, which is at least partially responsible for it timing out.

This is what we used to have: https://github.com/CenterForOpenScience/osf.io/commit/b6ce05f1a0d875f181fcad1e6b744caef9cb1ac9#diff-758f58f82207e3674b3105d2b3d4ce7a

**Pull request merged**